addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/check-launch-dependency/check-launch-dependency.js

```diff
@@ -227,6 +234,70 @@ class CheckLaunchDependency extends StepBase {
     });
   }
 
+  /**
+   * Method to allow ingress access from AppStream to ALB
+   *
+   * @param requestContext
+   * @param resolvedVars
+   * @param albDetails
+   */
+  async authorizeAlbAppStreamIngress(requestContext, resolvedVars, albDetails) {
```

Comment:
Any reason these methods aren't part of ALB CloudFormation?

Reply:
True, but only allowing ingress access from AppStream to ALB (AppStream -> ALB) can be done in ALB's CFN template. The egress access in AppStream SG for the ALB (ALB -> AppStream) will need to be granted/revoked programmatically.

I just preferred granting/revoking ingress and egress rules at the same time, thought the code is easier to maintain that way. Let me know if we want to split them.

Reply:
Moved ingress assignment to ALB template.